README.md

```diff
@@ -1 +1,41 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Project Name: Portfolio-setup-and-mobile-version-skeleton .
+
+> In this project I built the mobile version of my portfolio , It contains mainly two parts the header and the headline , the header is divided into two sections: the status bar 
+> which contains the clock,battery,wifi etc.. and the toolbar that have the logo section and the icon menu . The Headline contains 3 main section which introduce me and show how
+> to contact me. 
+
+
+
+In order to get a copy of this project :
+ - use The git clone command for this URl ( git clone [url] ) 
+     > https://github.com/Danieldotcomcoder/Portfolio-setup-and-mobile-version-skeleton.git
+ - Or visit the URL then click on code and Download ZIP.
+ - Use the live server extension of VS Code to preview the project.
+
+## Built With
+
+- Major languages :
+   * HTML — Hypertext Markup Language
+   * CSS — Cascading Style Sheet
+
+## Author
+
+👤 **Daniel Shdeed**
+
+- GitHub: [https://github.com/Danieldotcomcoder)]
+- Twitter: [https://twitter.com/DannyDotcoder)]
+- LinkedIn: [https://www.linkedin.com/in/daniel-shdeed-832b03115/]
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
# [OPTIONAL]
- [ ] It is always nice to acknowledge the design when you are using one, please consider using this section to acknowledge the figma design. 👍🏻